devito/operator/operator.py

```diff
@@ -894,14 +894,20 @@ def apply(self, **kwargs):
         >>> op = Operator(Eq(u3.forward, u3 + 1))
         >>> summary = op.apply(time_M=10)
         """
+        # Compile the operator before building the arguments list
+        # to avoid out of memory with greedy compilers
```

Comment:
is this related to

https://github.com/devitocodes/devito/issues/1374 

?

Reply:
Probably yes, tagging it